README.md

```diff
@@ -1,44 +1,18 @@
-# edgedelta-lambda-extension
-Edge Delta lambda extension to monitor AWS lambda functions.
+# Introduction
+Edge Delta Lambda Extension Layer to monitor AWS lambda functions. You can deploy this extension as a lambda layer by deploying ƒrom the AWS Serverless Application Repository. After you deploy the extension, you can add it to your lambda function as a layer to monitor your lambda function logs.
 
-To run this example, you will need to ensure that your build architecture matches that of the Lambda execution environment by compiling with GOOS=linux and GOARCH=amd64 if you are not running in a Linux environment.
-
-## Scripts
-
-We have helper scripts in the `scripts` folder to publish and remove lambda extension as a lambda layer version.
-The `publish.sh` script automates the commands in the following section.
-
-## Manuel Build
-
-Building and saving package into a bin/extensions directory:
+ARNs:
+- arn:aws:serverlessrepo:us-west-2:233765244907:applications/EdgeDelta-Lambda-Extension-Layer-AMD64
+- arn:aws:serverlessrepo:us-west-2:233765244907:applications/EdgeDelta-Lambda-Extension-Layer-ARM64
```

Comment:
I think we will not give them serverlessrepo arn, instead we will give them layer arn in the form of `arn:aws:lambda:<region>:<account>:layer:edgedelta-lambda-extension-<arch>:<version>` cc @asimerel